java/HealthVault Mobile/HVSample/src/main/res/layout/activity_main.xml

```diff
@@ -0,0 +1,237 @@
+<LinearLayout xmlns:android="http://schemas.android.com/apk/res/android"
```

Comment:
I have created task # 54619 for this comment

---
In reply to: [112366522](https://github.com/Microsoft/healthvault-samples/pull/4#discussion_r112366522) [](ancestors = 112366522)